pyinterprod/pronto/protein.py

```diff
@@ -289,3 +290,90 @@ def import_proteins(ora_url: str, pg_url: str):
 
     pg_con.close()
     logger.info("complete")
+
+
+def import_protein_pubmed(ip_url: str, swp_url: str, pg_url: str):
+    logger.info("populating protein2publication")
+    pg_con = psycopg.connect(**url2dict(pg_url))
+    with pg_con.cursor() as pg_cur:
+        pg_cur.execute("DROP TABLE IF EXISTS protein2publication")
+        pg_cur.execute(
+            """
+            CREATE TABLE protein2publication (
+                ACCESSION VARCHAR(15) NOT NULL,
+                PUBMED_ID VARCHAR(15) NOT NULL
```

Comment:
Store PubMed IDs as integer:

```suggestion
                PUBMED_ID INTEGER NOT NULL
```